src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_helpers.h

```diff
@@ -146,6 +146,29 @@ class RCHelpers {
    */
   static void RemoveRedundantGPSDataFromIVDataMsg(
       smart_objects::SmartObject& msg_params);
+
+  /**
+   * @brief MergeModuleData all keys and values from first parameter and
+   * update and append keys and values from the second
+   * @param data1 - initial data
+   * @param data2 - updated data
+   * @return updated data1 with data2 keys and values
+   */
+  static smart_objects::SmartObject MergeModuleData(
```

Comment:
@YarikMamykin please create two separate commits - one for refactoring and one for the fix

Reply:
@LitvinenkoIra , please take a look at 688641776f (fix) and 66a90ade0c (refactoring)